server/controllers/clubAccountControllers.js

```diff
@@ -11,15 +12,39 @@ module.exports = {
         // TODO
     },
     getProfile: function(req, res){
-        // TODO
+        clubProfile.find( {clubAccountId: req.params.id} )
```

Comment:
should we use the `find` directly in the clubProfile, or should we `findById` the account first, and then use the `clubProfileId` field in the account object to `findById` the profile?